README.md

```diff
@@ -66,6 +67,29 @@ Then to install buildah follow the steps in this example:
   buildah --help
 ```
 
+In Ubuntu/Debian, you can use this command:
+
+```
+  apt-get -qq update
+  apt-get install software-properties-common
+  add-apt-repository ppa:duggan/bats
+  add-apt-repository ppa:alexlarsson/flatpak
+  add-apt-repository ppa:gophers/archive
+  apt-get install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libostree-dev libseccomp-dev libselinux1-dev
+  apt-get install golang-1.8
```

Comment:
Probably wouldn't hurt to also add:

sudo apt-get dist-upgrade
sudo shutdown -r now